services/user_service/callbacks.py

```diff
@@ -0,0 +1,70 @@
+from constants import database_admin, pubsub_manager, RESULT_TOPIC
+import json
+
+
+def ack_message(message):
+    print(f"Received message : {message}")
+    print(f"Data : {message.data}")
+    message.ack()
+    return message.attributes["message_id"]
+
+
+def get_users_callback(message):
+    message_id = ack_message(message)
+    users = database_admin.get_all_users()
+    response = "There are no users" if len(users) == 0 else json.dumps(users)
```

Comment:
If json.dumps makes array, then it is better to pass empty json array instead of string